### PR TITLE
NEXUS-8362 repo status polling

### PR DESCRIPTION
**Pull request by kellyrob99**
https://issues.sonatype.org/browse/NEXUS-8362
screen shot 2015-04-06 at 10 46 09 pm
screen shot 2015-04-06 at 10 46 34 pm
screen shot 2015-04-06 at 10 48 36 pm

One thing to note is that status is not updated until an external request is processed for a repo and results in success/failure.

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RepositoryComponent.groovy [hunk 1]
```diff
@@ -101,25 +103,47 @@ extends DirectComponentSupport
...
         type: input.type,
         format: input.format,
         online: input.facet(ViewFacet).online,
-        status: status,
+        status: buildStatus(input),
         attributes: attributeConverter.asAttributes(input.configuration.attributes)
     )
   }
 
+  @DirectPollMethod(event = "coreui_Repository_readStatus")
```
Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RepositoryComponent.groovy]: JavaDoc

**Comment by adreghiciu** [comment]
+1

**Comment by cstamas** [comment]
+1

**Comment by kellyrob99** [comment]
http://bamboo.s/browse/NX3-OSSF156-2

**Pull request merged**